config/govuk_notify_templates.yml

```diff
@@ -9,7 +9,7 @@
 # we default to 'integration' group.
 #
 development:
-  citizen_start_application: 570e1b9d-6238-45fd-b75c-96f2f39db8e9
+  citizen_start_application: e747df7d-4842-4bca-88a6-4ad3ebd89994
```

Comment:
Should the production GUID change too?

Reply:
Yep that's in my left to do and will do that now, i just wanted to check that i was on the right track